src/units/unit.hpp

```diff
@@ -1741,13 +1741,13 @@ class unit : public std::enable_shared_from_this<unit>
 		return get_ability_bool(tag_name, loc_);
 	}
 
-	/** Checks whether this unit currently possesses a given ability used like weapon
+	/** Checks whether this unit currently possesses a given ability, and that that ability is active.
 	 * @return True if the ability @a tag_name is active.
-	 * @param special the const config to one of abilities @a tag_name checked.
+	 * @param ability the const config to one of abilities @a tag_name checked.
 	 * @param tag_name name of ability type checked.
 	 * @param loc location of the unit checked.
 	 */
-	bool get_self_ability_bool(const config& special, const std::string& tag_name, const map_location& loc) const;
+	bool get_self_ability_bool(const config& ability, const std::string& tag_name, const map_location& loc) const;
```

Comment:
The naming is inconsistent, with the argument that `get_self_ability_bool` and `get_adj_ability_bool` call `ability` being the same as `ability_active` and `ability_active`'s `cfg` argument, but those methods` `ability` argument being called `tag_name` here.

Agreed that it should be renamed from `special`, not sure what it should be renamed to. Maybe `wml_definition`?

Reply:
renamed to cfg and tag_name to ability